articles/understanding-dbms/index.md

```diff
@@ -0,0 +1,67 @@
+---
+layout: engineering-education
+status: publish
+published: true
+url: /engineering-education/understanding-dbms/
+title: Understanding Database Architecture and Models
+description: Learn the concepts of database and its three types of build architecture. Study the different aspects of the Hierarchical Data model, Network Data model and Relational Data model.
+author: kanishkvardhan-a-n
+date: 2020-09-21T00:00:00-05:00
+topics: []
+excerpt_separator: <!--more-->
+images:
+
+  - url: /engineering-education/understanding-dbms/hero.jpg
+    alt: Database Architecture and Models example image
+
+---
+Computers have made our lives easier in handling complicated calculations and solving complex problems. Hands down computers are one of the best inventions of mankind. While using computers and instilling them in our lives has made our work a piece of cake, maintaining them can be a difficult to accomplish.
+<!--more-->
+### Central Processing Unit
+Computers are nothing more but a series of electrical and electronic circuits that run together to form a single working unit called the **CPU (short for Central Processing Unit)**. One such component of the CPU is the **Memory Unit**.

```

Comment:
Please note that computers are more than electrical and electronic circuits that form a single unit that is called the CPU. A computer is a collection of parts, of which one of them is the CPU. You also have the motherboard, GPU, and power supply unit.

Reply:
Thank you @Manley12 . Glad to hear that you liked it.
That's a great point. I will make the necessary changes. Thanks for letting me know.